content/en/continuous_integration/guides/code_coverage.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Code coverage in Datadog
+kind: guide
+description: Learn how to report and use code coverage in Datadog
+further_reading:
+  - link: "/continuous_integration/tests"
+    tag: "Documentation"
+    text: "Exploring tests"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+
+## Report code coverage
+
+Before you can follow this guide, make sure that [Test Visibility][1] is already set up for your language.
+
+{{< tabs >}}
+{{% tab "Javascript/Typescript" %}}
+
+### Compatibility
+
+* `dd-trace>=3.20.0`.
+* `jest>=24.8.0`, only when run with `jest-circus`.
+* `mocha>=5.2.0`.
+* `cucumber-js>=7.0.0`.
+
+When tests are instrumented with [Istanbul][1], the Datadog Tracer reports code coverage under the `test.code_coverage.lines_pct` tag for your test sessions automatically.
+
+To instrument tests with Istanbul, the easiest way is to use [`nyc`][2]:
+
+```json
+{
+  "scripts": {
+    "test": "mocha",
+    "coverage": "nyc npm run test"
+  }
+}
+```
+
+Then use:
+
+```
+NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-service npm run coverage
+```
+
+Jest includes istanbul by default, so you can simply do:
+
+```json
+{
+  "scripts": {
+    "test:coverage": "jest --coverage"
+  }
+}
+```
+
+```
```

Comment:
Is there any instruction you'd like to provide here?

Reply:
yeah I guess I can use the same `Then, use`